projects/gsc/experiments/sampled_kwinners/sampled_kwinner_gsc_cnn.py

```diff
@@ -0,0 +1,168 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2019, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see http://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+import warnings
+
+from torch import nn
+
+from nupic.torch.modules import Flatten, SparseWeights, SparseWeights2d
+
+from .sampled_kwinners import SampledKWinners, SampledKWinners2d
+
+
+class SampledKWinnerGSCSparseCNN(nn.Sequential):
+    """Sparse CNN model used to classify `Google Speech Commands` dataset as
+    described in `How Can We Be So Dense?`_ paper.
```

Comment:
Maybe add a sentence saying it uses the sampled form of kwiners, just to distinguish this model from `GSCSparseCNN`.